### PR TITLE
Add form Section to the Portfolio

### PR DESCRIPTION
**Pull request by gak92**
In this part of the project I added:
Contact Form Section in index.html
Also, update the style.css for this section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -280,6 +281,60 @@ <h2>Skills</h2>
...
+                <textarea
+                  name="message"
+                  id="message"
+                  maxlength="500"
+                  cols="30"
+                  rows="5"
+                  placeholder="Write your message here"
+                  required
+                ></textarea>
+              </li>
```
Comment (koachcode) [inline index.html]: - Please there is an extra white line by the right of your webpage, that should not be there, please try as a must to follow the Figma design.

##### Figma design
Screenshot from 2022-05-31 20-29-39

##### Your design 
Screenshot from 2022-05-31 20-25-21

Please feel free to reach out to me if you need further, explanation of this feedback on slack, `@ kanu mike chibundu`

File: index.html [hunk 2]
```diff
@@ -280,6 +281,60 @@ <h2>Skills</h2>
...
+          <p class="contact-para">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+          <form
+            action="https://formspree.io/html/"
+            method="post"
+            class="contact-form"
+          >
```
Comment (koachcode) [inline index.html]: - Please check the action URL, I think it is missing the form_id you got from `freespree`, it makes your form not work or post the intended message, Please try to fix it and test it works on your end.

**Review by koachcode — CHANGES_REQUESTED** [review]
Hi @gak92 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
you-are-near

## Highlights
- Linter all green :+1: 
- Clean code :+1: 
- Professional PR nameing :100: 
- Good commit message :100: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — APPROVED** [review]
### Hi @gak92, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
> 
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**